### PR TITLE
NOISSUE - Update Provision service

### PR DESCRIPTION
**Pull request by rodneyosodo**
Signed-off-by: 0x6f736f646f <blackd0t@protonmail.com>

### What does this do?
It updates provision tools to use latest sdk config file

### Which issue(s) does this PR fix/relate to?
No

### List any changes that modify/break current functionality
Nothing

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
No

### Notes
N/A

### CODE REVIEW

File: tools/mqtt-bench/README.md [hunk 1]
```diff
@@ -47,6 +47,10 @@ Two output formats supported: human-readable plain text and JSON.
 Before use you need a `mfconn.toml` - a TOML file that describes Mainflux connection data (channels, thingIDs, thingKeys, certs).
 You can use `provision` tool (in tools/provision) to create this TOML config file.
 
+```bash
+./provision -u test@mainflux.com -p test1234 --host http://127.0.0.1 --num 100 > ../mqtt-bench/mfconn.toml
```
Comment (dborovcanin) [inline tools/mqtt-bench/README.md]: Use Go here and paths relative to Mainflux root:
```bash
go run tools/provision/cmd/main.go -u test@mainflux.com -p test1234 --host http://127.0.0.1 --num 100 > tools/mqtt-bench/mfconn.toml 
```

**Review by dborovcanin — CHANGES_REQUESTED** [review]

File: tools/mqtt-bench/README.md (hunk 1 above)
Comment (rodneyosodo) [reply tools/mqtt-bench/README.md]: Should this apply also to provision?

Comment (dborovcanin) [reply tools/mqtt-bench/README.md]: Let's do it like that.

**Review by dborovcanin — APPROVED** [review]

**Review by dborovcanin — CHANGES_REQUESTED** [review]
Let's fix the path in the `tools/provision/README.md`, as well.

**Review by dborovcanin — APPROVED** [review]

**Review by drasko — COMMENTED** [review]
LGTM

**Pull request merged**